components/Button/src/index.scss

```diff
@@ -4,7 +4,7 @@
   border: 0;
   border-radius: var(--denhaag-button-border-radius);
   color: var(--denhaag-button-primary-action-color);
-  cursor: pointer;
+  cursor: default;
```

Comment:
Do we have a source or explanation for why this is better from accessibility pov? Testing this in storybook I feel like it doesn't convey properly that it's a clickable element.

Reply:
I'll check this with Ferdinand, not sure why 'default' is better than 'pointer' in any case for a button.

Reply:
Asked the following to Fernand:

> Most of the buttons are simply links (<a href="[url]" class="denhaag-button"> ), should these also be given the default cursor? Or does this only apply to the situation <button class="denhaag-button"> ? What do you recommend in this?

And now we wait for a reply ;-) ( GDH-392 )

Reply:
Hi @bddjong , i've received response of Fernand. 

When it's a link with the button class it must show the pointer, otherwise it must show the default cursor.